src/zope/traversing/api.py

```diff
@@ -157,11 +120,9 @@ def getParent(obj):
 
 
 def getParents(obj):
-    """Returns a list starting with the given object's parent followed by
+    """
+    Returns a list starting with the given object's parent followed by
     each of its parents.
-
-    Raises a TypeError if the context doesn't go all the way down to
-    a containment root.
```

Comment:
~~That seems like useful information to me.~~

This seemed peculiar before I saw the interface `__doc__` copying at the end of the module.